modules/standard/Path.chpl

```diff
@@ -753,17 +893,25 @@ proc file.relPath(start:string=curDir): string throws {
    :type name: `string`
 */
  proc splitPath(name: string): (string, string) {
-   var rLoc, lLoc, prev: byteIndex = name.rfind(pathSep);
+   return splitPathImpl(name);
+ }
+
+ proc splitPath(name: bytes): (bytes, bytes) {
+   return splitPathImpl(name);
+ }
+
+ private proc splitPathImpl(name: ?t): (t, t) {
+   var rLoc, lLoc, prev = name.rfind(pathSep);
```

Comment:
Shouldn't this use the bytes version of pathSep in that case, e.g. `getPathSep(t)` ?